packages/lib/passport-interface/src/RequestTypes.ts

```diff
@@ -815,6 +815,25 @@ export interface FrogCryptoUpdateFeedsResponseValue {
   feeds: FrogCryptoDbFeedData[];
 }
 
+/*
+ * However, many problems can come up in {@link GenericIssuanceCheckInRequest}
+ * and {@link GenericIssuanceCheckRequest}. This type enumerates all the possible
+ * problems.
```

Comment:
this comment starting with `However,` is kinda funny haha

Reply:
Ah, it was copied from the older check-in code, and makes a bit less sense as a result.